tests/common_tests/helpers/tpcs_for_tests/v1/tp_model.py

```diff
@@ -45,8 +43,7 @@ def get_tp_model() -> TargetPlatformModel:
                              name='imx500_tp_model')
 
 
-def get_op_quantization_configs() -> \
-        Tuple[OpQuantizationConfig, List[OpQuantizationConfig], OpQuantizationConfig]:
+def get_op_quantization_configs() -> Tuple[OpQuantizationConfig, List[OpQuantizationConfig], OpQuantizationConfig]:
```

Comment:
generate_default_op_quantization_config

Reply:
I don't want to change the name of this function this would break to many tests and I don't think that naming is that important here